server/tests/meter/test_filter.py

```diff
@@ -26,6 +27,59 @@ def test_strip_metadata_prefix() -> None:
     assert clause.property == "property"
 
 
+class TestFilterClauseValueValidation:
+    """Test validation of FilterClause.value field to prevent database overflow."""
+
+    def test_valid_integer_values(self) -> None:
```

Comment:
Please use `@pytest.mark.parametrize` to generate those different cases. Docs: https://docs.pytest.org/en/6.2.x/parametrize.html